Protos/google/protobuf/compiler/plugin.proto

```diff
@@ -107,6 +107,16 @@ message CodeGeneratorResponse {
   // exiting with a non-zero status code.
   optional string error = 1;
 
+  // A bitmask of supported features that the code generator supports.
+  // This is a bitwise "or" of values from the Feature enum.
+  optional uint64 supported_features = 2;
```

Comment:
I am so happy about this!  It's long been a sore point that protoc had no way to know what a code generator could do.